data/appinfo.about.json

```diff
@@ -0,0 +1,13 @@
+{
+    "title": "Device Info",
+    "id": "org.webosports.app.settings.about",
+    "version": "0.4.0",
+	"vendor": "WebOS Ports",
```

Comment:
Can you cleanup the whitespaces here? Seems some are tab and some space. Also lower.